VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionService.java

```diff
@@ -69,7 +71,7 @@ public FollowBody(boolean following) {
 
 
     @GET("/api/discussion/v1/threads/")
-    TopicThreads getThreadList(@Query("course_id") String courseId, @Query("topic_id") String topicId, @Query("view") String view, @Query("order_by") String orderBy, @Query("page_size") int pageSize, @Query("page") int page ) throws RetroHttpException;
+    TopicThreads getThreadList(@Query("course_id") String courseId, @Query("topic_id") List<String> topicId, @Query("view") String view, @Query("order_by") String orderBy, @Query("page_size") int pageSize, @Query("page") int page ) throws RetroHttpException;
```

Comment:
Again I guess the parameter should be renamed to `topicIds` now, notwithstanding that the query parameter key is `topic_id`.